src/types/feeBreakdown.ts

```diff
@@ -0,0 +1,21 @@
+export interface FeeBreakdown {
+  amount: {
+    applicationFee: number;
+    total: number;
```

Comment:
nit: same feedback as previous PR - I'd personally find it easier to read/follow if we repeat `calculated`, `payable` here rather than introduce `applicationFee`, `total` !

Reply:
Please see [c0b81b5](https://github.com/theopensystemslab/planx-core/pull/578/commits/c0b81b50d56f97fbd0b73c7268025ed3105e27e4)